### PR TITLE
VIH-7637 vho unable to send messages remove email reference

### PR DESCRIPTION
**Pull request by alberto-hortiguela**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-7637

### Change description ###
Removing @hearings.reform.hmcts.net references from controller and unit test

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb.UnitTests/Hub/SendMessageTests.cs [hunk 1]
```diff
@@ -300,6 +305,14 @@ private void AssertMessageSentStatusToApi(string fromUsername, string toUsername
 
         private void SetupSendMessageTests()
         {
+            var configRootBuilder = new ConfigurationBuilder()
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb.UnitTests/Hub/SendMessageTests.cs:308]: Again no need to pull in user secrets here

File: VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs [hunk 2]
```diff
@@ -208,7 +210,7 @@ private async Task<bool> IsRecipientAdmin(string recipientUsername)
                 return true;
             }
 
-            if (!recipientUsername.EndsWith(ReformEmailDomain))
+            if (!recipientUsername.EndsWith(_servicesConfiguration.EmailReformDomain))
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs:213]: ignore case sensitivity

File: VideoWeb/VideoWeb.UnitTests/Hub/EventHubBaseTests.cs [hunk 3]
```diff
@@ -42,6 +46,9 @@ public void Setup()
             HubCallerContextMock = new Mock<HubCallerContext>();
             GroupManagerMock = new Mock<IGroupManager>();
             HeartbeatMapper = new Mock<IHeartbeatRequestMapper>();
+            ConferenceCacheMock = new Mock<IConferenceCache>();
+            ServicesConfigurationMock = new Mock<IOptions<HearingServicesConfiguration>>();
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb.UnitTests/Hub/EventHubBaseTests.cs:50]: Don't need to mock this.

File: VideoWeb/VideoWeb.UnitTests/Hub/SendMessageTests.cs [hunk 4]
```diff
@@ -300,6 +305,13 @@ private void AssertMessageSentStatusToApi(string fromUsername, string toUsername
 
         private void SetupSendMessageTests()
         {
+            var vhServicesConfigurationOptions = Options.Create(new HearingServicesConfiguration
+            {
+                EmailReformDomain = "@hearings.reform.hmcts.net"
+            });
+
+            AdminUsername = AdminUsername + vhServicesConfigurationOptions.Value.EmailReformDomain;
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb.UnitTests/Hub/SendMessageTests.cs:313]: why do we need the admin username set here?

File: VideoWeb/VideoWeb.UnitTests/Hub/EventHubBaseTests.cs [hunk 5]
```diff
@@ -42,6 +45,8 @@ public void Setup()
             HubCallerContextMock = new Mock<HubCallerContext>();
```
Comment (robertbennett1998) [inline VideoWeb/VideoWeb.UnitTests/Hub/EventHubBaseTests.cs:45]: Any chance you can change these tests to use AutoMock

**Review by robertbennett1998 — COMMENTED** [review]
Looks good, just address Shaeds comments and I will pick them up. Also, if you can convert the tests to AutoMock that would be great!

File: VideoWeb/VideoWeb.UnitTests/Hub/EventHubBaseTests.cs [hunk 6]
```diff
@@ -51,9 +58,18 @@ public void Setup()
             UserProfileServiceMock.Setup(x => x.GetObfuscatedUsernameAsync(It.IsAny<string>()))
                 .ReturnsAsync("o**** f*****");
 
-            ConferenceCacheMock = new Mock<IConferenceCache>();
+            var configRootBuilder = new ConfigurationBuilder()
```
[the remark this answers is not in the document]
Comment (alberto-hortiguela) [reply VideoWeb/VideoWeb.UnitTests/Hub/EventHubBaseTests.cs]: done

**Pull request closed without merging**